### DISCUSSION TITLE
Just installed i3wm for the first time on Mint, Mod+D and Mod+R don't work?

### DISCUSSION THREAD

**Discussion by RunielN597** [i3/i3#6078] [Q&A]
Pretty much the title. But I just started trying out i3wm and so far all I can do is open and close terminals. I can't launch applications by using Mod+D, resize using Mod+R, etc. The only combinations that work are opening and closing terminals, and switching between the windows.

**Comment by KebabLord** [comment]
Please provide your configuration so we can inspect better, also which version you are using? 

Mod+D relies on a 3rd party software called "dmenu" which you can install by `apt install suckless-tools`, make sure it's installed.

Also make sure that your mod key is set on configuration, i'm using "super" key as mod.
set $mod Mod4
 
 here is my configuration for resize:
 ```
 mode " Resize Mode" {
        bindsym Left        resize shrink width 5 px or 5 ppt
        bindsym Down        resize grow height 5 px or 5 ppt
        bindsym Up          resize shrink height 5 px or 5 ppt
        bindsym Right       resize grow width 5 px or 10 ppt
        bindsym Return mode "default"
        bindsym Escape mode "default"
        bindsym $mod+r mode "default"
}
```
